cloud/pkg/cloudhub/common/model/types.go

```diff
@@ -72,14 +72,15 @@ type HubInfo struct {
 
 // NewResource constructs a resource field using resource type and ID
 func NewResource(resType, resID string, info *HubInfo) string {
-	var prefix string
+	paths := make([]string, 4)
 	if info != nil {
-		prefix = fmt.Sprintf("%s/%s/", model.ResourceTypeNode, info.NodeID)
+		paths = append(paths, model.ResourceTypeNode, info.NodeID)
```

Comment:
`append` will still cause the slice to expand.
fix: 
```go
paths = make([]string, 0 , 4)
```

Reply:
Fixed, thx.